include/superior_mysqlpp/types/datetime.hpp

```diff
@@ -14,14 +14,33 @@
 
 namespace SuperiorMySqlpp
 {
+    /**
+     * @brief Class Datetime that works with date and time.
```

Comment:
Useless comment, please be more precise, describe intent of this class for example...

Reply:
ACK